src/DataFixtures/1000-talents/home.html

```diff
@@ -0,0 +1,85 @@
+<h2>Notre vision</h2>
+
+<p>
+    La vie politique de notre pays connait une profonde transformation. Nous avons la conviction profonde que
+    l'engagement de chacune et de chacun est un moteur puissant, indispensable à la vitalité de notre démocratie et à
+    l'émancipation individuelle et collective.
+</p>
+
+<p>
+    Or cet engagement est un parcours. Notre rôle est de guider celles et ceux qui souhaitent s’engager dans
+    l’action politique pour être utile à leur pays.
+</p>
+
+<p>
+    Le <b>programme 1000 talents</b> a pour mission de favoriser l'émergence et la montée en compétences de nouveaux
```

Comment:
For exemple, here it's the correct `'` char :+1:

Reply:
* For exAmple xD